incubator/fluentd/README.md

```diff
@@ -16,21 +16,20 @@ Switching backends requires using a matching image as well as some backend-speci
 | `elasticsearch.host` | `instance-name.region.es.amazonaws.com` |
 | `elasticsearch.port` | `443` |
 | `elasticsearch.scheme` | `https` |
-| `elasticsearch.extraArgs` | [] |
-| `elasticsearch.buffer_chunk_limit` | `5m` |
-| `elasticsearch.buffer_queue_limit` | `100` |
-| `elasticsearch.buffer_queue_full_action` | `exception` |
-| `elasticsearch.request_timeout` | `20s` |
+| `elasticsearch.chunk_limit_size` | `5m` |
+| `elasticsearch.chunk_limit_records` | `100` |
+| `elasticsearch.overflow_action` | `throw_exception` |
+| `elasticsearch.retry_timeout` | `20s` |
 | `elasticsearch.flush_interval` | `5s` |
-| `elasticsearch.num_threads` | `4` |
+| `elasticsearch.flush_thread_count` | `4` |
 
 
 ### Papertrail
 
 | Parameter | Default |
 | --------- | ------- |
 | `image.repository` | `fluent/fluentd-kubernetes-daemonset` |
-| `image.tag` | `v0.12.43-debian-papertrail` |
+| `image.tag` | `v1.4.2-debian-papertrail-1.1` |
```

Comment:
We should update the image.tag value under the list of elasticsearch parameters too right?